WebApplication/ClientApp/src/App.js

```diff
@@ -33,6 +36,13 @@ export class App extends Component {
   }
   componentDidMount() {
     this.props.fetchShowParametersChanged();
+
+    const params = window.location.search.substring(1);
+
+    if (params !== '') {
```

Comment:
I guess it is breaking application in normal usage for login scenario. Which we shouldn't even though we are not merging.

Reply:
Good point, fixed now :). The adoption now requires url parameter (?url=urlToJsonConfig)